recipe/meta.yaml

```diff
@@ -53,43 +51,60 @@ requirements:
     - {{ mpi }}
     - gmp
     - mpfr
-    - libhwloc
     - hdf5
     - hdf5 * {{ mpi_prefix }}_*
-    - tetgen
-    - fftw
     - libboost-headers
-    - eigen 
+    - eigen
     - voro
-    - cgal =5.6.1
-    - cgal-cpp =5.6.1
+    - cgal
+    - cgal-cpp
     - libaec
     - numpy
     - openssl
     - zlib
   run:
     - python
     - setuptools
-    - h5py 
+    - numpy
+    - h5py
     - gitpython
-    - pandas
-    - ifes-apt-tc-data-modeling
+    - jupyterlab
+    - jupyterlab-h5web
+    - nodejs
     - scikit-learn
+    - pandas
     - ase
-    - punx
-    - nexpy
-    - silx
+    - radioactivedecay
+    - ifes-apt-tc-data-modeling
+    - hdbscan
+    - sphinx
     - {{ pin_compatible('numpy') }}
+
+# pip install ecdf pytictoc ifes-apt-tc-data-modeling>=0.2.1 python-docs-theme xmltodict odfpy pylint
+
 test:
   imports:
+    # relevant for pure python tools
+    - paraprobe_utils
     - paraprobe_parmsetup
+    - paraprobe_reporter
     - paraprobe_transcoder
-  #commands:
+    - paraprobe_clusterer
+    # relevant for pure c/c++ tools
+    - paraprobe_ranger
```

Comment:
These are full C++ executable, therefore they should be listed under `commands`. Please see [this issue](https://gitlab.com/paraprobe/paraprobe-toolbox/-/issues/30) which was raised before.

Reply:
The CLI argument --help is possible with C/C++ executables for v0.5, paraprobe_clusterer, paraprobe_transcoder, paraprobe_nanochem, paraprobe_parmsetup are pure Python, otherwise comment implemented accordingly